cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/LedgerEvents/Gov/ProposeNewConstitution.hs

```diff
@@ -48,7 +48,7 @@ newtype AdditionalCatcher
 
 
 hprop_ledger_events_propose_new_constitution :: Property
-hprop_ledger_events_propose_new_constitution = H.integrationRetryWorkspace 2 "propose-new-constitution" $ \tempAbsBasePath' -> do
+hprop_ledger_events_propose_new_constitution = H.integrationWorkspace "propose-new-constitution" $ \tempAbsBasePath' -> do
```

Comment:
Given the CI's behavior the last days, this is bold :smile:

Reply:
IIRC this was due to the timeouts in our tests. These properties don't rely on timeouts.